src/test/java/com/google/api/codegen/testdata/discoveries/adexchangebuyer.v1.4.json.overrides

```diff
@@ -1,5 +1,8 @@
 {
```

Comment:
All these apiTypeName overrides were removed in #693, being no longer necessary.